ui/widgets/nuxeo-user-tag.js

```diff
@@ -56,6 +56,13 @@ import './nuxeo-tooltip.js';
           a:hover {
             @apply --nuxeo-link-hover;
           }
+
+          .user-name-clipped {
```

Comment:
I would suggest a more abstract name for this class, since in some cases it won't be clipping. Try to choose a name that describes the element instead.